<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hanoi</class>
 <widget class="QMainWindow" name="Hanoi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hanoi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushPEG1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPEG2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPEG3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>-10</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>651</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Middle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Right</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Game">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menu_Set_Disks">
     <property name="title">
      <string>Number of Disks</string>
     </property>
     <addaction name="action_2"/>
     <addaction name="action_3"/>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
     <addaction name="action_6"/>
     <addaction name="action_7"/>
     <addaction name="action_8"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="menu_Set_Disks"/>
   </widget>
   <widget class="QMenu" name="menuEXIT_GAME">
    <property name="title">
     <string>EXIT GAME</string>
    </property>
    <addaction name="actionEXIT"/>
   </widget>
   <addaction name="menu_Game"/>
   <addaction name="menuEXIT_GAME"/>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Undo_All">
   <property name="text">
    <string>Undo &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAuto_Play">
   <property name="text">
    <string>Auto &amp;Play</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>&amp;2</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>&amp;3</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>&amp;4</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>&amp;5</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>&amp;6</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>&amp;7</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>&amp;8</string>
   </property>
  </action>
  <action name="actionEXIT">
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionEXIT</sender>
   <signal>triggered()</signal>
   <receiver>Hanoi</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
